connect/src/routes/request.ts

```diff
@@ -68,6 +68,8 @@ export class RouteTransferRequest<N extends Network> {
       dq.warnings = [...quote.warnings];
     }
 
+    dq.eta = quote.eta;
```

Comment:
All the above props are set after empty checks. Do you think we need the same check for `eta`, or not necessary?

Reply:
I don't think so since quote.eta will either be a number or undefined, and if it's undefined then dq.eta will also be undefined. I don't think it's necessary unless you think we should change it

Reply:
I don't have a strong opinion about it, more for consistency as other props are set after a check. It's your call 👍